src/app/(private)/(routes)/(member)/forum/[postId]/[...titleSlug]/page.tsx

```diff
@@ -108,8 +108,8 @@ const PostPage = async ({ params }: PostPageProps) => {
               <div className="flex flex-col gap-1">
                 <span className="text-muted-foreground truncate text-lg">
                   <Link
-                    href="#"
-                    className="font-semibold hover:underline transition-all"
+                    href={`/user/${post?.user.username}`}
```

Comment:
What happens if `post` is undefined? We should have a fallback for this behavior.

Reply:
if (!post) {
    notFound()
  }